### ISSUE TITLE
Decide what to do with event_poll plugin

### ISSUE THREAD

**Issue by juho-jaakkola** [juho-jaakkola/elgg-event_calendar#27]
https://github.com/kevinjardine/event_poll

From the event_calendar point of view: is the dependency cleanly implemented, i.e. no issues with event_poll NOT being installed? Then it might be rather a question of testing event_poll's additional functionality (which could be done at a later time).

**Comment by juho-jaakkola** [comment]
It seems event_calendar works fine without event_poll.

I however greatly dislike the fact that event_poll is so deeply integrated into event_calendar. The correct way would be to use events and plugin hooks to add the poll features. (Possibly it wasn't possible back when the integration was done.)

**Comment by iionly** [comment]
How about just keeping it as it is for now (maybe adding event_poll as suggested plugin in manifest.xml)? And only once event_calendar and event_poll have been updated to fully work on Elgg 1.9/1.10 (only changing what needs to be changed to get it working) the way the integration is done can be improved.

**Comment by juho-jaakkola** [comment]
Ideal approach would be to allow a generic poll plugin to be integrated into event_calendar. E.g. the plugin that you're maintaining.

**Comment by juho-jaakkola** [comment]
Something like this would be even better: https://github.com/juho-jaakkola/elgg-scheduling

**Comment by iionly** [comment]
A generic poll plugin won't do. The event polls are very specific with the focus on scheduling events.

I don't know if your scheduling plugin will do either. The event_poll plugin offers already a lot more functionality - also due to the close connection with the event_calendar plugin. That's also the reason why I suggested to think about merging the event_poll plugin into event_calendar (and simply make the scheduling/poll functionality a plugin setting).

The only downside I currently see with the event_poll plugin is that its UI feels a bit unpolished. If you haven't tried it yet, you might want to make a little test run (https://github.com/iionly/event_poll) to see it in the works.

**Comment by juho-jaakkola** [comment]
The event_poll plugin has been added as suggested plugin in manifest.xml, so I'm closing this for now.
